packages/blockchain-link/src/workers/utils.ts

```diff
@@ -45,3 +46,23 @@ export const transformTarget = (target: VinVout, incoming: VinVout[]) => ({
     coinbase: target.coinbase,
     isAccountTarget: incoming.includes(target) ? true : undefined,
 });
+
+/**
+ * Sorts array of backend urls so the localhost addresses are first,
+ * then onion addresses and then the rest. Apart from that it will
+ * be shuffled randomly.
+ */
+export const prioritizeEndpoints = (urls: string[]) =>
+    urls
+        .map((url): [string, number] => {
+            const hostname = parseHostname(url);
+            let prio = Math.random();
```

Comment:
I guess `prio` here means `priority` ? Then why `prio` instead of `priority`? Usually full words are more readable and this case is not that long ...

Reply:
Weeeeeeeeeeeeeeeeeeeeell, https://github.com/trezor/trezor-suite/pull/5408/commits/360b1b953e587fa6c1d7bc1b2c4a90682b4f24bb

Reply:
Please please, use full words, I am super scared of 1-2 letter variables especially 🥺